PolyEngine/Engine/Src/CoreConfig.hpp

```diff
@@ -11,6 +22,19 @@ namespace Poly
 		bool DebugNormalsFlag = false;
 		bool WireframeRendering = false;
 		bool DisplayFPS = true;
+
+		Dynarray<String> GetAssetsPaths(eResourceSource Source) const;
+
+		bool GetIsLoadedFromFile() const;
```

Comment:
If getter returns bool it's nicer to name it starting with "Is". "GetIs" just looks akward.